src/Components/DataBatchContext.tsx

```diff
@@ -35,6 +35,15 @@ export const DataBatchContextProvider = connect(
       setLimit(configuredLimit)
     }
 
+    const key = [
+      'DataBatchContextProvider',
+      content?.id(),
+      attribute,
+      id,
+      tag,
+      search,
+    ].join('-')
```

Comment:
* [ ] FYI functionality-wise this works nicely. Unfortunately a reader has no idea why this works and which changes are wanted/unwanted. For example, (why) can('t) I add `limit` or remove `search`? Ideas:
  * Just use `search` as the key. This is more self-explanatory than a seemingly random collection.
  * Add a comment.
  * Extract to a helper or variable with an explanatory name.